src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -190,6 +195,38 @@ def get_inventory_script(inventory):
     return inventory
 
 
+def get_callback_plugin_dir(test_env):
+    """Determine callback plugins path, create if missing, set env. var."""
+    callback_plugin_dir = os.path.join(
+        os.environ["TOX_WORK_DIR"], "callback_plugins"
+    )
+    if not os.path.isdir(callback_plugin_dir):
+        os.makedirs(callback_plugin_dir)
+    test_env["ANSIBLE_CALLBACK_PLUGINS"] = callback_plugin_dir
```

Comment:
Is it safe to clobber the variable? Or does an existing value need to be kept and only get appended to?

Reply:
> Is it safe to clobber the variable? Or does an existing value need to be kept and only get appended to?

I suppose if someone were trying to run with their own custom callback plugin e.g.

```
ANSIBLE_CALLBACK_PLUGINS=/path/to/my_great_plugin tox -e qemu-ansible-core-2.17 ...
```

then that would not work

Reply:
Ah, so these are supposed to be exclusive, ok.